modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -104,6 +108,8 @@
 
 			&-text {
 				width: 100%;
+				display: var( --n-tabs-title-text-display );
```

Comment:
I am thinking if we always use center and flex, then we don't need a css variable. I think you can style this without css variables.

Reply:
Done